### PR TITLE
Replace Guava by JDK (Partly)

### PR DESCRIPTION
**Pull request by pks-1981**
Replace Guava by JDK (Partly)
https://github.com/wiremock/wiremock/issues/2111

## Submitter checklist

- [x] The PR request is well described and justified, including the body and the references
- [x] The PR title represents the desired changelog entry
- [x] The repository's code style is followed (see the contributing guide)
- [x] Test coverage that demonstrates that the change works as expected
- [ ] For new features, there's necessary documentation in this pull request or in a subsequent PR to [wiremock.org](https://github.com/wiremock/wiremock.org)

### CODE REVIEW

**Comment by pks-1981** [comment]
@tomakehurst @oleg-nenashev
please look MR how it will be possible

File: src/main/java/com/github/tomakehurst/wiremock/common/ContentTypes.java [hunk 1]
```diff
@@ -18,12 +18,13 @@
 import static com.github.tomakehurst.wiremock.common.Strings.stringFromBytes;
 import static com.github.tomakehurst.wiremock.common.TextType.JSON;
 import static java.util.Arrays.asList;
+import static java.util.Map.entry;
+import static java.util.Map.ofEntries;
```
Comment (oleg-nenashev) [inline src/main/java/com/github/tomakehurst/wiremock/common/ContentTypes.java:22]: A little bit unusual to do such an import, but okay I guess

File: src/main/java/com/github/tomakehurst/wiremock/core/WireMockApp.java [hunk 2]
```diff
@@ -423,7 +416,7 @@ public FindServeEventsResult removeServeEventsForStubsMatchingMetadata(
 
   @Override
   public FindNearMissesResult findNearMissesForUnmatchedRequests() {
-    ImmutableList.Builder<NearMiss> listBuilder = ImmutableList.builder();
+    List<NearMiss> listBuilder = new ArrayList<>();
```
Comment (oleg-nenashev) [inline src/main/java/com/github/tomakehurst/wiremock/core/WireMockApp.java:419]: Rename the variable to avoid confusion?

**Review by oleg-nenashev — APPROVED** [review]
Looks good overall

File: src/main/java/com/github/tomakehurst/wiremock/core/WireMockApp.java (hunk 2 above)
Comment (pks-1981) [reply src/main/java/com/github/tomakehurst/wiremock/core/WireMockApp.java:419]: Renamed to nearMisses

File: src/main/java/com/github/tomakehurst/wiremock/common/ContentTypes.java (hunk 1 above)
Comment (pks-1981) [reply src/main/java/com/github/tomakehurst/wiremock/common/ContentTypes.java:22]: Remade without static

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**